test/rician.jl

```diff
@@ -0,0 +1,97 @@
+@testset "Rician" begin
+
+    d1 = Rician(0.0, 10.0)
+    @test d1 isa Rician{Float64}
+    @test params(d1) == (0.0, 10.0)
+    @test shape(d1) == 0.0
+    @test scale(d1) == 200.0
+    @test partype(d1) === Float64
+    @test eltype(d1) === Float64
+    @test rand(d1) isa Float64
+
+    d2 = Rayleigh(10.0)
+    @test mean(d1) ≈ mean(d2)
+    @test var(d1) ≈ var(d2)
+    @test mode(d1) ≈ mode(d2)
+    @test median(d1) ≈ median(d2)
+    @test quantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90]) ≈ quantile.(d2, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test pdf.(d1, 0.0:0.1:1.0) ≈ pdf.(d2, 0.0:0.1:1.0)
+    @test cdf.(d1, 0.0:0.1:1.0) ≈ cdf.(d2, 0.0:0.1:1.0)
+
+    d1 = Rician(10.0, 10.0)
+    @test median(d1) == quantile(d1, 0.5)
+    x = quantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test all(cdf.(d1, x) .≈ [0.25, 0.45, 0.60, 0.80, 0.90])
+
+    x = rand(Rician(5.0, 5.0), 100000)
+    d1 = fit(Rician, x)
+    @test d1 isa Rician{Float64}
+    @test params(d1)[1] ≈ 5.0 atol=1e-1
```

Comment:
It seems this fails with Julia nightly since the error is 0.12 (https://github.com/JuliaStats/Distributions.jl/pull/1387/checks?check_run_id=3503702946#step:6:647). I assume it's just due to a different `x`. Can you relax the test a bit?

Reply:
That's strange. `runtests.jl` sets a random number seed to make these tests repeatable:
https://github.com/JuliaStats/Distributions.jl/blob/ed52948fb47e7e0aecaf81e99ebb090c2738cf0e/test/runtests.jl#L76

And on my machine, the tests pass. Anyway I've now added a seed while generating random numbers to see if it fixes the problem:
`x = rand(MersenneTwister(3456789), Rician(5.0, 5.0), 100000)`

Can you  check if the tests pass now?

Reply:
Random numbers are not guaranteed to be reproducible with different Julia versions and operating systems, even if a seed is fixed.

Reply:
So specifying the RNG explicitly here does not guarantee that the tests pass. StableRNG.jl fixes this issue but often the simpler solution is just to use a slightly larger tolerance.

Reply:
I see, I didn't realize that. Increased tolerance now -- hopefully that fixes this?

Reply:
These tests pass now